cypress/integration/admin/pageBuilder/blocks/pageImport.cy.js

```diff
@@ -0,0 +1,71 @@
+function makeid(length) {
```

Comment:
I know we discussed this, but when I think about it again, let's use [nanoid](https://github.com/ai/nanoid/) here. Let's use it instead of the `makeid` we introduced here and in other test files.

Before using it, add the lib by running the following cmd from project root:

```bash
 yarn add "nanoid@^3.0.0" --dev 
```